### PR TITLE
[MCH-244-111] refactor: three pool crv solver [MCH-243]

### PR DESCRIPTION
**Pull request by gotzenx**
Refactor to match `crvYfiEth` and `crvSpellEth` solvers structure

### CODE REVIEW

**Review by b0dhidharma — APPROVED** [review]
Nice ! Thanks for taking care of this sir. 😍

File: packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { SimpleEnabledTrade, Solver } from '@scripts/libraries/types';
 import * as wallet from '@test-utils/wallet';
 import { ethers } from 'hardhat';
 
+const DUST_THRESHOLD = utils.parseEther('1');
```
Comment (poolpitako) [inline packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts:10]: not sure if this should be part of the `Solver` interface so developers are forced to set a threshold which makes sense. 1 ether in 3crv is 1.02 usd.

**Review by poolpitako — APPROVED** [review]

File: packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts (hunk 1 above)
Comment (b0dhidharma) [reply packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts:10]: I mean, the thing is: what should the heuristic be here ? Profitability with respects of % of fees won by Yearn by harvest?

Comment (b0dhidharma) [reply packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts:10]: Clearly the 1 ether amount is just a placerholder rn.

Comment (poolpitako) [reply packages/yswaps/scripts/libraries/solvers/multicall/ThreePoolCrv.ts:10]: something to tackle in another issue or pr for sure. :+1:

**Pull request merged**